test/integration/docker-linux-baseline/controls/tests.rb

```diff
@@ -0,0 +1,48 @@
+include_controls 'linux-baseline' do
+  # /etc/hosts.equiv is not absent everywhere
+  skip_control 'os-01'
+  # skip entropy test, as our short living test VMs usually do not have enough
+  skip_control 'os-08'
+  # skip filesystem test (vfat enabled for uefi)
+  skip_control 'os-10'
+  # No syslog installed in test env
+  skip_control 'package-07'
+  # skip auditd tests, we do not have any implementation for audit management yet
+  skip_control 'package-08'
+
+  # docker environment - skip all sysctl tests
```

Comment:
@mcgege Hm. This is strange, the https://github.com/dev-sec/linux-baseline/pull/91/files should basically do the auto-detection here and exclude specific tests for container environments. Did you have any issues with it?

Reply:
Yes, had some problems here, but I cannot remember the details right now :-)
Have to recheck this ...